src/components/ReviewTransaction.test.tsx

```diff
@@ -145,3 +153,107 @@ describe('ReviewDetailsItem', () => {
     expect(tree.getByTestId('BoldItem/Label')).toHaveStyle(typeScale.labelSemiBoldMedium)
   })
 })
+
+describe('ReviewTotalValue', () => {
+  const celoToken = mockTokenBalances[mockCeloTokenId] as unknown as TokenBalance
+  const cUSDToken = mockTokenBalances[mockCusdTokenId] as unknown as TokenBalance
+  it.each([
+    {
+      tokenInfo: { ...celoToken, priceUsd: new BigNumber(1) },
+      tokenAmount: new BigNumber(10),
+      localAmount: new BigNumber(10),
+      feeTokenInfo: undefined,
+      feeTokenAmount: undefined,
+      feeLocalAmount: null,
+      title:
+        'returns token and local amounts only for send operation if there is no fee but local price is available',
```

Comment:
```suggestion
        'returns the token and fiat amount when fee info is missing',
```
is there any need to mention the "send operation" here and in the tests below, for this ReviewTransactionTotal component? Are we intending to need to add tests when using this for swap and earn?

Reply:
Ah, you're right! I wanted to point out that it will only consider token for _the_ operation and as I was working on Send I just put it there without thinking. I'll adjust!
It also needs a part with `but local price is available` cause there are two scenarios: with/without fiat.